packages/manager/modules/pci/src/projects/new/payment/components/credit/index.html

```diff
@@ -0,0 +1,62 @@
+<div class="pci-project-new-payment-credit">
+    <p data-translate="pci_project_new_payment_credit_explain"></p>
+
+    <div class="d-flex justify-content-between mb-3">
+        <oui-radio
+            class="oui-radio_btn"
+            data-ng-repeat="amount in $ctrl.predifinedAmounts track by $index"
+            data-name="credit"
+            data-thumbnail
+            data-variant="light"
+            data-model="$ctrl.model.credit"
+            data-value="amount"
+            data-on-change="$ctrl.onPredifinedAmountRadioChange()"
+            data-disabled="$ctrl.globalLoading.finalize"
+        >
+            <span data-ng-bind="amount.text"></span>
+        </oui-radio>
+        <oui-radio
+            class="oui-radio_btn"
```

Comment:
You have to add this extra clss on the component ?

Reply:
Yes. On models provided by CX, the radio is not visible